src/box/memtx_engine.cc

```diff
@@ -523,11 +523,14 @@ memtx_engine_commit(struct engine *engine, struct txn *txn)
 	struct txn_stmt *stmt;
 	stailq_foreach_entry(stmt, &txn->stmts, next) {
 		if (stmt->add_story != NULL || stmt->del_story != NULL) {
-			ssize_t bsize = memtx_tx_history_commit_stmt(stmt);
+			int64_t ctuples = 0;
+			ssize_t bsize =
+				memtx_tx_history_commit_stmt(stmt, &ctuples);
```

Comment:
Nit: you can pass `mspace->compressed_tuples` to this function directly, without using the temporary `ctuples` variable.

Reply:
fixed